src/scene/vcModel.cpp

```diff
@@ -395,9 +395,13 @@ void vcModel::ContextMenuListModels(vcState *pProgramState, vdkProjectNode *pPar
       {
         vcModel *pOldModel = (vcModel *)pChildNode->pUserData;
         const double ballRadius = 0.15; // TODO: Expose this to the user
-        udWorkerPoolCallback callback = [this, pProgramState, pOldModel, ballRadius](void*)
+
+        int SRID = (this->m_pPreferredProjection ? this->m_pPreferredProjection->srid : (pOldModel->m_pPreferredProjection ? pOldModel->m_pPreferredProjection->srid : -1));
+        SRID = (SRID == -1 ? (this->m_pCurrentZone ? this->m_pCurrentZone->srid : (pOldModel->m_pCurrentZone ? pOldModel->m_pCurrentZone->srid : -1)) : SRID);
```

Comment:
The point of the comparison feature is to duplicate the model and just add an attribute for displacement. This code should only be getting the SRID that is stored in the UDS, not an override value, nor the `pOldModel` value.